syncano/models/fields.py

```diff
@@ -687,13 +686,52 @@ def to_native(self, value):
         return value
 
 
-GeoPointStruct = namedtuple('GeoPointHelper', ['latitude', 'longitude'])
+class GeoPoint(object):
 
+    def __init__(self, latitude, longitude):
+        self.latitude = latitude
+        self.longitude = longitude
 
-class GeoPoint(Field):
+    def __str__(self):
```

Comment:
I'm pretty sure if you specify `__repr__` - it will be used for `__str__` as well. Keep it DRY.